### PR TITLE
Pract.01: Add 02-04 | Ivan Stoynev

### PR DESCRIPTION
**Pull request by Vacata12**
Ivan Stoynev
3 6MI0700164

### CODE REVIEW

File: Seminars/Sem.01/Pract.01/Task03.cpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+int**  createMatrix(size_t row, size_t col);
+void inputArr(int** arr, size_t row, size_t col);
+void printArr(int** arr, size_t row, size_t col);
+int** transponeMatrix(int** arr, size_t row, size_t col);
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/Task03.cpp:5]: Missing const here.

File: Seminars/Sem.01/Pract.01/task04.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+struct Student
+{
+    char name[30];
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/task04.cpp:5]: 30 + 1 due to null teminator?

File: Seminars/Sem.01/Pract.01/task04.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    unsigned int factNum;
+};
+
+void initStudent(Student st);
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/task04.cpp:9]: reference , not by copy

File: Seminars/Sem.01/Pract.01/task04.cpp [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+void printStudent(Student st);
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/task04.cpp:10]: const reference, not by copy

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]

**Comment by Justsvetoslavov** [comment]
Closing as we have merged solutions

File: Seminars/Sem.01/Pract.01/task04.cpp (hunk 4 above)
Comment (Vacata12) [reply Seminars/Sem.01/Pract.01/task04.cpp:10]: Ok, thanks

**Pull request closed without merging**